pkg/provisioningv2/rke2/planner/encryptionkeyrotation.go

```diff
@@ -0,0 +1,450 @@
+package planner
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	rkev1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
+	"github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1/plan"
+	"github.com/rancher/rancher/pkg/controllers/provisioningv2/rke2"
+	"github.com/rancher/wrangler/pkg/generic"
+	"github.com/sirupsen/logrus"
+	capi "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+const (
+	Start             = "start"
```

Comment:
Given that these constants are under the planner package, can we rename them to indicate that they're for encryption key rotation to help prevent confusion?